web/src/i18n/locales/en.yaml

```diff
@@ -308,7 +308,7 @@
         subheader: "The condition described at § 433.112(b)(18) requires that states submit mitigation plans addressing strategies to reduce the consequences of failure for all major milestones and functionality. Maintaining a mitigation plan is an industry standard best practice for any major IT build; CMS expects that states will identify potential risks and develop strategies to address those risks throughout the system lifecycle. Mitigation plans for major Medicaid E&E system or MMIS projects should address minimum expected functionality, critical success factors, and risk factors as tied to major milestones identified in the APD. The mitigation plans should also reflect key events and dates that would trigger the mitigation, and projected timeframe for the mitigation sunset. States should consider strategies to address risks for the M&O phase of the project, such as staffing issues and software upgrades. CMS expects states to revise and resubmit their plans to CMS as risks and mitigations change along the system life cycle. States’ proposed mitigations should be commensurate with the nature and scope of the identified risks. We also expect that states will submit their mitigation plans with the APDs to demonstrate compliance with this condition."
         title: "Describe how you'll meet the Mitigation Strategy condition"
       keyPersonnel: 
-        header: "Key Personnel"
+        header: Personnel"
```

Comment:
Key Personnel is actually the correct term for this citation. Looks like the "Key (with quote) was removed.

Reply:
Will fix.